src/mame/drivers/renegade.cpp

```diff
@@ -524,15 +540,14 @@ void renegade_state::renegade(machine_config &config)
 	SPEAKER(config, "mono").front_center();
 
 	GENERIC_LATCH_8(config, m_soundlatch);
-	m_soundlatch->data_pending_callback().set_inputline(m_audiocpu, M6809_IRQ_LINE);
+	m_soundlatch->data_pending_callback().set_inputline(m_audiocpu, M6809_IRQ_LINE, ASSERT_LINE);
```

Comment:
AFAIK data_pending_callback does assert/clear automatically. It auto clears when soundlatch is read. So, the trampoline and manually clearing irq isn't needed either

Reply:
Thanks, I meant to check if that was the case! I will revert...